### DISCUSSION TITLE
How to change page render results script id prefix ~ ?

### DISCUSSION THREAD

**Discussion by forwells** [vikejs/vike#916] [Discussion]
![image](https://github.com/brillout/vite-plugin-ssr/assets/32950945/bcf5afb5-2d97-44dc-9b1b-ba9195f16bb0)
Here is the id start with vite-plugin-ssr
how can I change it to a custom string~

**Comment by brillout** [comment]
You can't.

Why do you want to change it?

**Comment by forwells** [comment]
We hope to hide the key information of the class library used by the key basic technology stack in the project, the text shown here is too easy to recognize😀

**Reply by brillout** [reply]
That's an unusual wish ;-). Why do you want to do this?
